src/techonomy/django/django_tableselect/columns.py

```diff
@@ -13,6 +14,8 @@
 
 
 class CheckBoxColumn(BaseCheckBoxColumn):
+    header_template = "django_tableselect/bulk_checkbox.html"
```

Comment:
Suggestie: Je selecteert een bulk, maar het je selecteert alles. 

Dus misschien "select_all_checkboxes.html" ? In plaats van bulk_checkbox?

Reply:
Klinkt logisch, ga ik veranderen!